### ISSUE TITLE
Example to reconstruct full order book from updates

### ISSUE THREAD

**Issue by elyase** [soulmachine/crypto-crawler-rs#11]
It would be great to have an example on how to keep a full order book from updates.
[Related gateio example ](https://github.com/gateio/gatews/blob/master/python/examples/local_order_book.py)

**Comment by soulmachine** [comment]
Each `OrderBookMsg` has a `seq_id` and a `prev_seq_id`, see [msg.rs#L107-L109](https://github.com/crypto-crawler/crypto-crawler-rs/blob/main/crypto-msg-parser/src/msg.rs#L107-L109),  if there are gaps, which means some messages are missing.

I'll submit an example for Binance soon.

**Comment by elyase** [comment]
that would be great! I am curious to know what other people are doing

**Comment by xtuyaowu** [comment]
> Each `OrderBookMsg` has a `seq_id` and a `prev_seq_id`, see [msg.rs#L107-L109](https://github.com/crypto-crawler/crypto-crawler-rs/blob/main/crypto-msg-parser/src/msg.rs#L107-L109), if there are gaps, which means some messages are missing.
> 
> I'll submit an example for Binance soon.

dose the example submited? thanks

**Comment by soulmachine** [comment]
@xtuyaowu No, there is no example for now. You can read the ["How to manage a local order book correctly"](https://binance-docs.github.io/apidocs/futures/en/#diff-book-depth-streams) section from Binance
